src/test/java/com/scalar/database/verification/GetTotal.java

```diff
@@ -33,7 +33,14 @@ public static void main(String[] args) throws IOException {
     TransferContext context = new TransferContext(numAccounts, 0, 0, 0);
     AccountBalanceTransferHandler handler = new AccountBalanceTransferHandler(config, context);
 
-    List<Result> results = handler.readRecordsWithRetry();
+    List<Result> results = null;
+    try {
+      results = handler.readRecordsWithRetry();
```

Comment:
When the exception is thrown and propagated to the main, doesn't this script return 1 ?

Reply:
No, the script just hangs... That does seem pretty weird doesn't it.